bigbluebutton-html5/imports/api/users/server/modifiers/userInactivityInspect.js

```diff
@@ -0,0 +1,34 @@
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/users';
+
+export default function userInactivityInspect(userId, responseDelay) {
+  check(userId, String);
+  check(responseDelay, Match.Integer);
+
+  const selector = {
+    userId,
+    inactivityCheck: false,
```

Comment:
Not sure why we need to filter for `inactivityCheck: false,` - userId should suffice. Are there cases where we don't want to update the user?

Reply:
Just to avoid an unnecessary update, but it's not a big deal.